renderdoc/driver/vulkan/vk_counters.cpp

```diff
@@ -547,13 +547,22 @@ struct VulkanKHRCallback : public VulkanDrawcallCallback
   void PreDispatch(uint32_t eid, VkCommandBuffer cmd) override { PreDraw(eid, cmd); }
   bool PostDispatch(uint32_t eid, VkCommandBuffer cmd) override { return PostDraw(eid, cmd); }
   void PostRedispatch(uint32_t eid, VkCommandBuffer cmd) override { PostRedraw(eid, cmd); }
-  void PreMisc(uint32_t eid, DrawFlags flags, VkCommandBuffer cmd) override { PreDraw(eid, cmd); }
+  void PreMisc(uint32_t eid, DrawFlags flags, VkCommandBuffer cmd) override
+  {
+    if(flags & DrawFlags::PassBoundary)
```

Comment:
This skipping should also be added to `VulkanAMDDrawCallback`.

Reply:
Done, I missed that somehow before.